samples/libcurl/src/main/kotlin/org/konan/libcurl/CUrl.kt

```diff
@@ -46,13 +54,14 @@ fun header_callback(buffer: CPointer<ByteVar>?, size: size_t, nitems: size_t, us
     return size * nitems
 }
 
-/*
-fun write_callback(buffer: COpaquePointer?, size: size_t, nitems: size_t, userdata: COpaquePointer?): size_t {
+
+fun write_callback(buffer: CPointer<ByteVar>?, size: size_t, nitems: size_t, userdata: COpaquePointer?): size_t {
     if (buffer == null) return 0
+    val data = buffer.toKString((size * nitems).toInt()).trim()
```

Comment:
Makes sense to put it inside `if`.

Reply:
Hello
Not sure if I follow.

Can you tell me what change you want me to do? And will update the PR